java-frontend/src/test/java/org/sonar/java/resolve/SymbolTableTest.java

```diff
@@ -873,8 +873,9 @@ public void constructorWithTypeArguments() {
     JavaSymbol.TypeJavaSymbol classSymbol = (JavaSymbol.TypeJavaSymbol) result.symbol("MyClass");
     List<JavaSymbol> constructors = classSymbol.members.lookup("<init>");
 
-    assertThat(constructors.get(0).usages()).hasSize(1);
-    assertThat(constructors.get(1).usages()).hasSize(1);
+    // FIXME : there should be one reference to each constructor : this is a bug in how method inference specificity is applied see JLS8 18.5.4
```

Comment:
This is bothering me. By fixing this ticket we may fix a case (and obviously fix an issue which was there for a while), but we break another one which was working... And this test was existing solely for the case which is now not properly resolved. Should we reoppen [SONARJAVA-1667](https://jira.sonarsource.com/browse/SONARJAVA-1667)?